Gemfile

```diff
@@ -61,6 +62,7 @@ end
 
 group :development do
   gem 'guard-rspec', require: false
+  gem 'guard-shell', require: false
```

Comment:
seems like this is specific to a particular dev setup and not needed in the Gemfile. 

> dependencies of a developer utility should not be able to interfere with the dependencies of your app.

https://github.com/ddollar/foreman/pull/437#issuecomment-41110407